core/src/test/scala/integration/kafka/admin/RemoteTopicCrudTest.scala

```diff
@@ -473,6 +473,40 @@ class RemoteTopicCrudTest extends IntegrationTestHarness {
     verifyRemoteLogTopicConfigs(newProps)
   }
 
+  @ParameterizedTest
+  @ValueSource(strings = Array("kraft"))
+  def testUpdateThreadPoolSize(quorum: String): Unit = {
```

Comment:
This test succeeds with and without the fix. The issue is while applying the dynamic configuration and this test does not cover that scenario.

Reply:
I forgot that the broker only logs error message when dynamic configuration is broken, so this test can pass even without the fix. Replaced it with unit test. Thanks.